site/docs/features/management.md

```diff
@@ -5,34 +5,107 @@ service can be scheduled and Nessie reports the outcome of each scheduled operat
+
+To be marked as unreferenced an asset must either:
+ 1. No longer referenced by any branch or tag. For example, an entire branch was deleted, and a table on that branch is no longer accessible.
+ 2. Assets created in a commit which has passed the (configurable) commit age. *If they are not referenced by newer commits*
+
+Identifying unreferenced assets is a non-destructive action. The result of the spark job is a Spark DataFrame of all the
+unreferenced assets. This dataframe is stored in an iceberg table managed by nessie at a configurable key. This table is
+referencable in Nessie so can be examined via Spark or any other Nessie/Iceberg compatible engine. An example of the table output
+is shown below.
+
+![Screenshot](../img/gc-table.png)
+
+This action is designed to run concurrently to other workloads and can/should be run regularly. This table is used as input
+into the destructive GC operation described below.
+
+#### Configuration and running
+GcActionsConfig actionsConfig, GcOptions gcConfig, TableIdentifier table
+The relevant configuration items are:
+| parameter | default value | description |
+|---|---|---|
+| table | `null` | The Iceberg `TableIdentifier` to which the unreferenced assets should be written |
+| GcOptions.getBloomFilterCapacity | 10000000 | Size of bloom filter for identification of referenced values |
+| GcOptions.getMaxAgeMicros | 7 days | age at which a commit starts to expire |
+| GcOptions.getTimeSlopMicros | 1 day | minimum age a values can be before it will be considered expired |
+| GcActionsConfig.getDynamoRegion | provider default | AWS Region of the Nessie DynamoDB |
+| GcActionsConfig.getDynamoEndpoint | provider default | Custom AWS endpoint of the Nessie DynamoDB |
+| GcActionsConfig.getStoreType | DYNAMO | only backend which supports GC |
+
+Running the action can be done simply by:
+```java
+    GcActions actions = new GcActions.Builder(spark)
+                                     .setActionsConfig(actionsConfig)
+                                     .setGcConfig(gcOptions)
+                                     .setTable(TABLE_IDENTIFIER).build(); // (1)
+    Dataset<Row> unreferencedAssets = actions.identifyUnreferencedAssets(); // (2)
+    actions.updateUnreferencedAssetTable(unreferencedAssets); // (3)
+```
+The first step above builds the action with known configs. Step 2 generates a DataFrame of unreferenced assets and
+Step 3 writes it as an iceberg table.
+
+### Delete Unreferenced Assets
+
+The destructive garbage collection step is also a Spark job and takes as input the table that has been built above. This job
+is modelled as an Iceberg Action and has a similar API to the other Iceberg Actions. In the future it will be registered with
+Iceberg's Action APIs and callable via Iceberg's custom [SQL statements](http://iceberg.apache.org/spark-procedures/).
+
+This Iceberg Action looks at the generated table from the 'Identify' step and counts the number of times a distinct asset has been
```

Comment:
Can you add a link to `Identify Unreferenced Assets` ?

Reply:
fixed